plugins/airtable/src/airtable.ts

```diff
@@ -412,71 +529,36 @@ export async function syncTable({
     return result
 }
 
-/*
- * Given a set of Airtable field schemas, returns a list of possible
- * fields that can be used as slugs.
- */
-export function getPossibleSlugFields(fieldsSchema: AirtableFieldSchema[]) {
-    const options: AirtableFieldSchema[] = []
-
-    for (const fieldSchema of fieldsSchema) {
-        switch (fieldSchema.type) {
-            case "singleLineText":
-                options.push(fieldSchema)
-        }
+function getIgnoredFieldIds(rawIgnoredFieldIds: string | null) {
+    if (!rawIgnoredFieldIds) {
+        return []
     }
 
-    return options
-}
-
-function getSuggestedFieldsForTable(tableSchema: AirtableTableSchema, ignoredFieldIds: string[]) {
-    const fields: ManagedCollectionField[] = []
-
-    for (const fieldSchema of tableSchema.fields) {
-        if (ignoredFieldIds.includes(fieldSchema.id)) continue
-
-        const field = getCollectionForAirtableField(fieldSchema)
-        if (field) {
-            fields.push(field)
-        }
-    }
+    const parsed = JSON.parse(rawIgnoredFieldIds)
```

Comment:
Should try/catch any JSON parsing, because if some dodgy data somehow gets into `rawIgnoredFieldIds`, it will be stuck in a fail loop

Reply:
Nice catch!